src/library/filescanner.c

```diff
@@ -1852,11 +1903,58 @@ playlist_add_files(FILE *fp, int pl_id, const char *virtual_path)
 	break;
 
       DPRINTF(E_DBG, L_SCAN, "Item '%s' added to playlist (id = %d)\n", dbmfi.path, pl_id);
+      ++added;
     }
 
   db_query_end(&qp);
   free(qp.filter);
 
+  path = NULL;
+  while (ret == 0 && added == 0 && strncasecmp(virtual_path, "/http:/", strlen("/http:/")) == 0)
+    {
+      // if not found, check in current playlist
```

Comment:
This should not be necessary anymore after the change to the virtual path for streams. The virtual path for streams should now be identical to the path (plus the leading '/'), or am i missing something?